tattoo_shop/static/description/index.html

```diff
@@ -0,0 +1,35 @@
+<h2><strong>Odoo for Tattoo Shops</strong></h2>
+<p>This industry package includes all the essential features to manage your tattoo shop, from customer relationship management to consent form signatures and appointments.</p>
+<h3><strong>Basics</strong></h3>
+<ul>
+    <li>
+        Use the <strong><font class="text-o-color-1">CRM App</font></strong> to manage your leads and track ongoing tattoo projects.
+    </li>
+    <li>
+        Handle consent forms and important documents effortlessly with the <strong><font class="text-o-color-1">Sign App</font></strong>.
+    </li>
+    <li>
+        Showcase your work, attract new clients through the <strong><font class="text-o-color-1">Website App</font></strong>.
+    </li>
+    <li>
+        Let your customers book and manage appointments through the <strong><font class="text-o-color-1">Appointments App</font></strong>.
+    </li>
+    <li>
+        Share your latest creations and engage with your audience using the <strong><font class="text-o-color-1">Social Marketing App</font></strong>.
+    </li>
+    <li>
+        Restock supplies with ease using the <strong><font class="text-o-color-1">Purchase App</font></strong>.
+    </li>
+    <li>
+        Manage your opportunities through  <strong><font class="text-o-color-1">Activities</font></strong>.
+    </li>
+</ul>
+<h3><strong>Customisations</strong></h3>
+<ul>
+    <li>
+        Added "Tags" as field on the website contact form.
+    </li>
+    <li>
+        Renamed website contact form fields to be more focused.
+    </li>
```

Comment:
I feel like this is not really what we call customization. I'll have a note on the task.